### PR TITLE
[place] chart title updates

### PR DESCRIPTION
**Pull request by beets**
* Updated titles of the charts based on our discussion today
* Added explicit unit / scaling factor for related charts
* Added support for "grams" as a unit

### CODE REVIEW

File: server/chart_config.json [hunk 1]
```diff
@@ -243,13 +262,19 @@
       "RetailDrugDistribution_DrugDistribution_Morphine"
     ],
     "isOverview": true,
+    "unit": "g",
     "relatedChart": {
+      "title": "Prescribed Drugs Per Capita",
       "scale": true,
-      "denominator": "Count_Person"
+      "denominator": "Count_Person",
+      "unit": "g"
```
Comment (shifucun) [inline server/chart_config.json:270]: very nice!

File: static/js/place/chart_block.tsx [hunk 2]
```diff
@@ -108,14 +120,9 @@ class ChartBlock extends React.Component<ChartBlockPropType, unknown> {
...
-                dcid={this.props.dcid}
-                placeType={this.props.placeType}
-                chartType={chartType}
                 snapshot={this.props.data.child}
-                title={`${this.props.data.title} across states within ${this.props.placeName}`}
-                unit={unit}
-                names={this.props.names}
-                scaling={scaling}
+                title={`${relatedChartTitle}: states within ${this.props.placeName}`}
+                {...sharedProps}
```
Comment (shifucun) [inline static/js/place/chart_block.tsx:125]: wow, this is cool

**Review by shifucun — DISMISSED** [review]
nice change!

**Comment by beets** [comment]
thanks! fixed the webdriver test

**Review by shifucun — APPROVED** [review]

**Pull request merged**